numpy/core/fromnumeric.py

```diff
@@ -679,10 +679,11 @@ def partition(a, kth, axis=-1, kind='introselect', order=None):
 
     Creates a copy of the array with its elements rearranged in such a
     way that the value of the element in k-th position is in the
-    position it would be in a sorted array. All elements smaller than
-    the k-th element are moved before this element and all equal or
-    greater are moved behind it. The ordering of the elements in the two
-    partitions is undefined.
+    position it would be in a sorted array.  In the partitioned array, all
```

Comment:
"it" is a bit ambiguous, the element or its value.  I think "value" would be correct, i.e., the same as the value of the k-th element of the sorted array.

Reply:
Thanks @charris, I pushed an update.  I also fixed `kth` to `k-th` to match the existing usage.